tests/perf-regression/perf-regression.json

```diff
@@ -0,0 +1,102 @@
+{
+  "sha256": {
+    "compileTime": 2.445895783000626,
```

Comment:
So these measurements were obtained by your machine? Will this differ a lot with CI / other machines?